core/domain/exp_jobs_one_off.py

```diff
@@ -59,7 +59,7 @@
 ALLOWED_IMAGE_EXTENSIONS = list(itertools.chain.from_iterable(
     feconf.ACCEPTED_IMAGE_FORMATS_AND_EXTENSIONS.values()))
 FILE_MODEL_ID_REGEX = re.compile(
-    r'^/([^/]+)/assets/(([^/]+)\.(' + '|'.join(
+    r'^/([^/]+)/assets/image/(([^/]+)\.(' + '|'.join(
```

Comment:
Just wondering, should this match GCS_IMAGE_ID_REGEX fully?

More generally, the more parity between GCS and local datastore we could have, the better. Ideally anything outside fs_domain has no idea what filesystem it's writing to, it should all just behave the same way.